src/main/java/nextstep/subway/path/domain/PathFinder.java

```diff
@@ -0,0 +1,63 @@
+package nextstep.subway.path.domain;
+
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.line.domain.Section;
+import nextstep.subway.line.domain.Sections;
+import nextstep.subway.path.exception.PathException;
+import nextstep.subway.station.domain.Station;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+import java.util.List;
+
+public class PathFinder {
+    private WeightedMultigraph<Station, DefaultWeightedEdge> graph;
+    private DijkstraShortestPath<Station, DefaultWeightedEdge> dijkstraShortestPath;
+
+    public PathFinder() {
+        graph = new WeightedMultigraph(DefaultWeightedEdge.class);
+        dijkstraShortestPath = new DijkstraShortestPath(graph);
```

Comment:
```suggestion
        graph = new WeightedMultigraph<>(DefaultWeightedEdge.class);
        dijkstraShortestPath = new DijkstraShortestPath<>(graph);
```
사소한 차이지만 꼭 raw타입으로 인식되지 않도록 해주세요 :)
코드 정합성을 지키는데 도움이 됩니다.